src/modules/elements/month/element.js

```diff
@@ -22,7 +22,8 @@ import './style.pcss';
 const today = new Date();
 const currentYear = today.getFullYear();
 const currentMonth = today.getMonth();
-const fromMonth = new Date( currentYear, currentMonth );
+const yearsBack = 5;
```

Comment:
might be a good idea to use a uppercase name to indicate the presence of a "constant" like on PHP side like. `YEARS_BACK`

Reply:
I agree, though everything here has previously done in lower case, so I'm sticking with that convention.